packages/datadog-plugin-openai/test/index.spec.js

```diff
@@ -0,0 +1,2110 @@
+              expect(traces[0][0]).to.have.property('type', 'openai')
+              expect(traces[0][0]).to.have.property('resource', 'createCompletion')
+              expect(traces[0][0]).to.have.property('error', 0)
+              expect(traces[0][0].meta).to.have.property('openai.request.method', 'POST')
+              expect(traces[0][0].meta).to.have.property('openai.request.endpoint', '/v1/completions')
+
+              expect(traces[0][0].meta).to.have.property('component', 'openai')
+              expect(traces[0][0].meta).to.have.property('openai.api_base', 'https://api.openai.com/v1')
+              expect(traces[0][0].meta).to.have.property('openai.organization.name', 'kill-9')
+              expect(traces[0][0].meta).to.have.property('openai.request.model', 'text-davinci-002')
+              expect(traces[0][0].meta).to.have.property('openai.request.prompt', 'Hello, ')
+              expect(traces[0][0].meta).to.have.property('openai.request.stop', 'time')
+              expect(traces[0][0].meta).to.have.property('openai.request.suffix', 'foo')
+              expect(traces[0][0].meta).to.have.property('openai.request.user', 'hunter2')
+              expect(traces[0][0].meta).to.have.property('openai.response.choices.0.finish_reason', 'length')
+              expect(traces[0][0].meta).to.have.property('openai.response.choices.0.logprobs', 'returned')
+              expect(traces[0][0].meta).to.have.property('openai.response.choices.0.text', 'FOO BAR BAZ')
+              expect(traces[0][0].meta).to.have.property('openai.response.model', 'text-davinci-002')
+              expect(traces[0][0].meta).to.have.property('openai.user.api_key', 'sk-...ESTS')
+              expect(traces[0][0].metrics).to.have.property('openai.request.best_of', 2)
+              expect(traces[0][0].metrics).to.have.property('openai.request.echo', 0)
+              expect(traces[0][0].metrics).to.have.property('openai.request.frequency_penalty', 0.11)
+              expect(traces[0][0].metrics).to.have.property('openai.request.logit_bias.50256', 30)
+              expect(traces[0][0].metrics).to.have.property('openai.request.logprobs', 3)
+              expect(traces[0][0].metrics).to.have.property('openai.request.max_tokens', 7)
+              expect(traces[0][0].metrics).to.have.property('openai.request.n', 1)
+              expect(traces[0][0].metrics).to.have.property('openai.request.presence_penalty', -0.1)
+              expect(traces[0][0].metrics).to.have.property('openai.request.temperature', 1.01)
+              expect(traces[0][0].metrics).to.have.property('openai.request.top_p', 0.9)
+              expect(traces[0][0].metrics).to.have.property('openai.response.usage.completion_tokens', 16)
+              expect(traces[0][0].metrics).to.have.property('openai.response.usage.prompt_tokens', 3)
+              expect(traces[0][0].metrics).to.have.property('openai.response.usage.total_tokens', 19)
```

Comment:
Could probably do `expect(traces[0][0]).to.deep.equal({ ... })` to make all this a bit cleaner. 🤔